README.md

```diff
@@ -1,2 +1,4 @@
 # family-bundle-calculator
-vanilla
+vanilla JS
+
+Horizon Wireless is the Unofficial mobile phone sponsor for all Forza Horizon events.
```

Comment:
lol nice

Reply:
Ha, all sorts of sponsors in the world of racing...